pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -39,26 +47,29 @@ if command -v apt-get >/dev/null 2>&1; then
 		fi
 	fi
 elif command -v dnf >/dev/null 2>&1; then
+	if ! command -v tar >/dev/null 2>&1; then
+		dnf_install -y tar
+	fi
 	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ]; then
 		if ! command -v sshfs >/dev/null 2>&1; then
-			if grep -q "release 8" /etc/system-release; then
-				dnf install --enablerepo powertools -y fuse-sshfs
+			if grep -q "release 8" /etc/system-release && grep -qv "^Oracle" /etc/system-release; then
```

Comment:
Why not move this to `dnf_install()` ?

Reply:
How would you do that? Add `--enablerepo powertools` to **all** dnf install command that are not on Oracle?

Maybe that is fine, but then you need to comment that this is really only done for `fuse-sshfs`?

Reply:
Yes

Reply:
This has a separate conditional due to the `--enablerepo powertools` part. That's not valid for Oracle Linux. If it's `release 8` and not Oracle, powertools is needed, if it's `release 8` and Oracle, ol8_codeready_builder is needed, otherwise we can just install sshfs directly. Combined with needing to keep repo flags limited for Oracle Linux due to repo/dnf performance, this seemed the best way. I could see an arguement for doing the same repo filtering for all rhel8 like linux releases though, so if you want, I can move that conditional into the dnf_install function.